qa/suites/deepsea/tier4/upgrade/3-test-phase.yaml

```diff
@@ -1,9 +1,37 @@
+overrides:
+        deepsea:
+                repositories:
+                        - name: ses
+                          url: http://10.86.0.120/artifacts/ci/d4ad635e5108134c42c5e5e477a4224a320ca2a57e0795865712f7f2f7ebf1ec/SUSE-Enterprise-Storage-6-POOL-x86_64-Build5.6
+                        - name: basesystem
+                          url: http://10.86.0.120/artifacts/ci/e9bf06179ab8222426cd8b22e7e5d63f40d2b12f67d6a47a4898983f5735ccdc/SLE-15-Module-Basesystem-POOL-x86_64-Build668.1
+                        - name: server
+                          url: http://10.86.0.120/artifacts/ci/618743a8bcd28a64b2881bd11eaccb24f17fe887576e109f59a64e46f9564f0a/SLE-15-Module-Server-Applications-POOL-x86_64-Build668.1
+                        - name: internal
+                          url: http://10.86.0.120/artifacts/ci/55d6cc29675bc77b0b44bd3063b830547a1f271e8e42c96e4729acf1dadb0209/SUSE-Enterprise-Storage-6-POOL-Internal-x86_64-Build5.2
+                        - name: update
+                          url: http://10.86.0.120/artifacts/ci/snapshot/201811060915/3577b41f35e5000f7bd8aef985ed8b32cbeb766d07cb7efbdbb8234e935d3789/SLE-15-Update
```

Comment:
this bit of YAML will be dropped - the person running the test (or Jenkins) will have to provide this bit of yaml via the command line, with the URLs of the latest milestone/version of SES6